scapy/fields.py

```diff
@@ -1234,6 +1234,18 @@ def i2repr(self, pkt, x):
         return bytes_hex(x[:self.length_from(pkt)]).decode()
 
 
+class XLEStrLenField(XStrLenField):
+    def i2repr(self, pkt, x):
+        a = bytes_hex(x[:self.length_from(pkt)]).decode()
+        return hex(int("0x" + a, 16))
```

Comment:
It doesn't make any sense to have `0x` in front of a hex encoded bytes-string.

It does with numbers, but we have `lhex()` to handle it in that case.